logstash-core/src/test/java/org/logstash/plugins/pipeline/PipelineBusTest.java

```diff
@@ -0,0 +1,166 @@
+package org.logstash.plugins.pipeline;
```

Comment:
I split tests across here and some rspec tests. The rspec tests are more comprehensive in terms of functionality.